rskj-core/build.gradle

```diff
@@ -27,6 +27,21 @@ jacocoTestReport {
     }
 }
 
+task quickTest(type: Test) {
```

Comment:
Maybe we can update the name and use parallelTest instead of quick.

Reply:
Addressed